src/components/custom/_card.tsx

```diff
@@ -54,9 +54,15 @@ const SelectedIcon = styled.img`
 `
 
 const Card = ({ icon, title, description, selected }: CardProps) => {
-    const [is_selected, setIsSelected] = React.useState(selected || false)
+    const [is_selected, setIsSelected] = React.useState(false)
     const clickHandler = () => setIsSelected(true)
 
+    React.useEffect(() => {
+        if (selected) {
```

Comment:
do we need this if condition here? 
initially the state will be `false` so there will be no changes.